src/Applications/GEOSdas_App/testsuites/x3dvar_ladas.input

```diff
@@ -44,7 +44,7 @@ FVHOME? [/discover/nobackup/qzhang/x3dvar_ladas]
 The directory /discover/nobackup/qzhang/x3dvar does not exist. Create it now? [y]
 > 
 
-Processing nodes (1:Haswell, 2:Skylake, 3:Cascase, 4:Milan)? [2]
+Processing nodes (3:Cascade, 4:Milan)? [3]
```

Comment:
this may need further tweaks.  maybe milan should be the default

Reply:
this example  3dvar setup input file should be replaced by a hybrid4dnvar input file based on x050 setup.

Reply:
FYI, I just merged the latest develop into the PR's branch, which required resolving a merge conflict for the above lines.  I'm a bit unsure about the defaults, but I'm fairly confident it doesn't matter at this moment.  I mostly wanted to make sure the branch still builds after merging the latest develop